.github/workflows/ci.yml

```diff
@@ -0,0 +1,34 @@
+name: Continuous Integration
```

Comment:
This is basically a cut down version from Axiom - functional but we might want to integrate Ruff instead?